### PR TITLE
Removed 'depth' parameter recommendation, updated README and CHANGELOG

### PR DESCRIPTION
**Pull request by vladimirbazhanov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,15 +93,17 @@ brew bundle
 
 ### Bootstrap application
 
-1. Clone application as new project with original repository named "rails-base". We use depth parameter here in order not to copy the history of changes in base project
+1. Clone application as new project with original repository named "rails-base" and delete `.git` directory to remove history of changes in base project.
 
    ```bash
-   git clone --depth 1 git://github.com/fs/rails-base.git --origin rails-base [MY-NEW-PROJECT]
+   git clone git://github.com/fs/rails-base.git --origin rails-base [MY-NEW-PROJECT]
+   rm -rf [MY-NEW-PROJECT]/.git
```
Comment (timurvafin) [inline README.md]: If you will remove `git` completely we will lose ability to pull changes from upstream (rails-base)

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Removed recommendation to clone with 'depth -1' key
```
Comment (timurvafin) [inline CHANGELOG.md]: Please add here link to original discussion

**Comment by vladimirbazhanov** [comment]
@timurvafin Now it's only removes --depth 1 and adds description in changelog, pls review

**Comment by vladimirbazhanov** [comment]
bump)
does we need this still?

**Comment by timurvafin** [comment]
@vladimirbazhanov please update PR

**Comment by timurvafin** [comment]
@vladimirbazhanov please reopen when will be updated.

**Pull request closed without merging**